src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -429,14 +430,19 @@ internal static int NamedPipeBufferSizeForRemoting
         /// Creates a RemoteSessionNamedPipeServer with the current process and AppDomain information.
         /// </summary>
         /// <returns>RemoteSessionNamedPipeServer.</returns>
-        public static RemoteSessionNamedPipeServer CreateRemoteSessionNamedPipeServer()
+        internal static RemoteSessionNamedPipeServer CreateRemoteSessionNamedPipeServer()
         {
             string appDomainName = NamedPipeUtils.GetCurrentAppDomainName();
 
-            return new RemoteSessionNamedPipeServer(NamedPipeUtils.CreateProcessPipeName(
+            return CreateRemoteSessionNamedPipeServer(NamedPipeUtils.CreateProcessPipeName(
```

Comment:
It's better to revert this change. It adds one additional method call without much value.

Reply:
fixed.